{
  "ngDeps": {
    "1": "angular.core.facade.exceptions",
    "3": [
      {
        "1": "base_wrapped_exception.dart",
        "5": false
      },
      {
        "1": "exception_handler.dart",
        "5": false
      }
    ],
    "4": [
      {
        "1": "exception_handler.dart"
      }
    ],
    "6": "exceptions.dart",
    "10": [
      {
        "1": "base_wrapped_exception.template.dart",
        "4": "i0"
      },
      {
        "1": "exception_handler.template.dart",
        "4": "i1"
      }
    ]
  },
  "type": {
    "BaseException": {
      "class": "Identifier",
      "name": "BaseException",
      "moduleUrl": "asset:angular2/lib/src/facade/exceptions.dart",
      "prefix": null,
      "constConstructor": null,
      "value": null
    },
    "WrappedException": {
      "class": "Identifier",
      "name": "WrappedException",
      "moduleUrl": "asset:angular2/lib/src/facade/exceptions.dart",
      "prefix": null,
      "constConstructor": null,
      "value": null
    },
    "makeTypeError": {
      "class": "Identifier",
      "name": "makeTypeError",
      "moduleUrl": "asset:angular2/lib/src/facade/exceptions.dart",
      "prefix": null,
      "constConstructor": null,
      "value": null
    },
    "unimplemented": {
      "class": "Identifier",
      "name": "unimplemented",
      "moduleUrl": "asset:angular2/lib/src/facade/exceptions.dart",
      "prefix": null,
      "constConstructor": null,
      "value": null
    },
    "_ArrayLogger": {
      "class": "Identifier",
      "name": "_ArrayLogger",
      "moduleUrl": "asset:angular2/lib/src/facade/exception_handler.dart",
      "prefix": null,
      "constConstructor": null,
      "value": null
    },
    "ExceptionHandler": {
      "class": "Identifier",
      "name": "ExceptionHandler",
      "moduleUrl": "asset:angular2/lib/src/facade/exception_handler.dart",
      "prefix": null,
      "constConstructor": null,
      "value": null
    }
  },
  "alias": {}
}